Rakefile

```diff
@@ -23,4 +41,4 @@ task :sync_tool do
   FileUtils.cp "../ruby/tool/lib/find_executable.rb", "./test/lib"
 end
 
-task :default => :test
+task :default => [:test, :test_in_isolation]
```

Comment:
Also I'm not so sure about testing the same thing twice in `default` task. The output and the testing time seem redundant for local execution. We could catch corner cases on CI, and it would hopefully not happen too often once we add enough `require`s.